src/Diagnostics/StdDiagnostics/StdDiagnostics.jl

```diff
@@ -0,0 +1,59 @@
+"""
+    StdDiagnostics
+
+This module defines many standard diagnostic variables and groups that may
+be used directly by experiments.
+"""
+module StdDiagnostics
+
+using CLIMAParameters
+using CLIMAParameters.Planet
+using CLIMAParameters.Atmos
+using CLIMAParameters.SubgridScale
+
+using KernelAbstractions
+using MPI
+using OrderedCollections
+using Printf
+using StaticArrays
+
+using ..Diagnostics # temporarily
```

Comment:
```suggestion
using ..Diagnostics
```
I think you have a note for this, but just a friendly reminder